### DISCUSSION TITLE
can't set chunk options globally

### DISCUSSION THREAD

**Discussion by cmusso86** [quarto-dev/quarto-web#393] [Q&A]
Hello, I am opening a discussion because I am not sure if it is an issue of if I am not finding the information in the documentation (maybe I am not looking in the right places). But since Quarto also uses knitr , I thought I could use knitr::opts_chunk$set() function to set the option for the chunks.
 At first I thought it was working ok (it is not showing the code, for example, as I wanted), but I also set it to not show messages and warnings and they keep showing in the pdf unless I set it in each chunk individually... 
so I don't know if I am doing something wrong... 
Thanks everybody in advance!

**Comment by mcanouil** [comment]
See <https://quarto.org/docs/computations/r.html> and <https://quarto.org/docs/computations/execution-options.html>.

Also next time, you might want to post on <https://github.com/quarto-dev/quarto-cli> repository 😉
